.github/workflows/generate-artifacts.yml

```diff
@@ -0,0 +1,69 @@
+name: allcontributors-zenodo
```

Comment:
what is the purpose of this workflow? is it an example to be added by people to the corresponding projects and used in this repo as an example and for "eat your own dog food" ?
I do use github workflows artifacts but find it not really convenient to use to fetch those artifacts (UI is still "clunky", artifacts are packaged into archive etc).  We might thought provide a helper `a2z apply-artifact` which would fetch the most recent for current branch and extract...  I like bots (submitting a PR) approach better but it would be a bigger undertaking I guess

Reply:
This is how I test GitHub actions.

Reply:
FWIW -- might be worth making use of ninja here -- provide a template (not just an example) and then a command to populate/update this one.  I do that in https://github.com/datalad/datalad-extensions/ where https://github.com/datalad/datalad-extensions/blob/master/templates/.github/workflows/test-%7B%7Bextension.name%7D%7D.yaml provides a template, so all the test-datalad* in https://github.com/datalad/datalad-extensions/tree/master/.github/workflows are generated based on metadata from https://github.com/datalad/datalad-extensions/blob/master/extensions.yaml .  Otherwise it is a copy which needs to be kept in sync with current state of example/code etc

Reply:
I think jinja2 is overkill for such a simple json structure. It's an extra dependency that we don't need to add.